cmd/gravity/commands/ledger.go

```diff
@@ -87,6 +87,11 @@ var (
 				ChainType:              "waves",
 				GravityContractAddress: "3PLpMu2cAg618e7xXYHtckFJjFZksPFHoLm",
 			},
+			account.Ergo.String(): {
+				NodeUrl: "http://10.10.10.4:9016",
```

Comment:
10.* seems like a internal IP

Reply:
It's right
I will close this PR and after the final update and final test, I will send the PR again.